### PR TITLE
Add linux directory to examples

### PR DESCRIPTION
**Pull request by stuartmorgan**
## Description

When Linux support was added to these plugins, the app template wasn't
yet stabilized, so was not checked in. Now that it is stable, this adds
them so that the plugin_tools workaround that created them on each run
can be removed.

Other than CHANGELOG.md updates and updating the verison in
pubspec.yaml, this is purely the result of running 'flutter create .' in
the example/ folders and adding the resulting linux/ directories.

## Related Issues

None

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by stuartmorgan** [comment]
Ping? This should be largely a rubber-stamp review, and lets us get rid of now-unnecessary workarounds in the plugin repo tooling. (I'll have a Windows version as well, but wanted to keep them separate to reduce PR size.)

**Review by cyanglaz — APPROVED** [review]
LGTM

File: packages/path_provider/path_provider/example/linux/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+flutter/ephemeral
```
Comment (cyanglaz) [inline packages/path_provider/path_provider/example/linux/.gitignore:1]: linux directory in a non-linux plugin.

**Review by cyanglaz — CHANGES_REQUESTED** [review]

File: packages/path_provider/path_provider/example/linux/.gitignore (hunk 1 above)
Comment (stuartmorgan) [reply packages/path_provider/path_provider/example/linux/.gitignore:1]: `path_provider` supports Linux; this isn't a non-Linux plugin.

**Review by cyanglaz — APPROVED** [review]
LGTM

**Pull request merged**